modules/viewing-pipeline-artifacts.adoc

```diff
@@ -0,0 +1,32 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-artifacts_{context}"]
+= Viewing pipeline artifacts
+
+[role='_abstract']
+When a pipeline run executes, you can view its pipeline artifacts from the {productname-short} dashboard. Pipeline artifacts can help you evaluate the performance of your pipeline runs and make it easier to understand your pipeline components. Pipeline artifacts can range from plain text data or detailed, interactive data visualizations.  
```

Comment:
Suggest replacing 'When' with 'After', if artifacts are only available for executed runs.

Suggest adding 'to' before 'evaluate'.

Suggest updating 'or' to 'to' in the last sentence (can range from x to y):

Pipeline artifacts can range from plain text data to detailed, interactive data visualizations.

Reply:
Changed to your suggestions.